roles/convert/tasks/main.yml

```diff
@@ -6,4 +6,7 @@
     reboot_timeout: 120
   timeout: 180
   when: convert_reboot_requested
+
+- name: Remove Convert2RHEL
+  ansible.builtin.include_tasks: remove-c2r.yml
```

Comment:
I was thinking maybe we shouldn't remove it if the conversion was unsuccessful referencing this new PR https://github.com/oamg/infra.convert2rhel/pull/9.
What do you think?

Reply:
I think it's fine to remove it if unsuccessful because the common role will install it anyway when run again.

Reply:
For now, I think it's totally fine to remove it every time. We could discuss this later and remove only when the conversion is successful to save some time and network, but right now, I believe this doesn't matter too much, and we can safely remove whenever we want.